test/recipes/30-test_evp.t

```diff
@@ -119,6 +121,8 @@ my @defltfiles = qw(
                      evppkey_rsa.txt
                     );
 push @defltfiles, qw(evppkey_brainpool.txt) unless $no_ec;
+push @defltfiles, qw(evppkey_ecdsa_rfc6979.txt) unless $no_ec || $no_ec2m;
```

Comment:
Could you please split the ec2m testcases into a separate file?

I'll approve then.

Reply:
Hmm, actually $no_ec2m should not be necessary. The evp_test should handle missing support for a key type gracefully. Could you please drop it?

Reply:
It actually crashed with that line removed.. 
I am a bit suprised this has happened before now.. Basically the "Key=XXX" lines fails.. so it sets t->skip = 1.. And then it continues to parse every line of the test (And NULL pointer access in then possible).
I have changed it so it skips the rest of the parsing in this case - I could check for NULL, but it seems silly to continue parsing a skipped test.